### PR TITLE
Add Imprint screen, Fix small searchfield issue

### PR DESCRIPTION
**Pull request by me-jterhorst**
Finished the Imprint Screen, 

Individualised the scrollbar experience ( tested on Edge, Chrome, Firefox and Chrome Mobile) 

Fixed small Searchbar issue that came up

### CODE REVIEW

File: client/src/pages/Imprint.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          so by logging in and visiting your overview, or account tab. Should
+          you wish to remove the data, that is associated with your account, you
+          can do so by simply deleting your account from the account page. Once
+          the Account delete has been issued you will be redirected to the
+          homescreen and all saved reminders will be lost.
+        </p>
+      </Card>
+    </main>
+  );
+}
```
Comment (patrikglanemann) [inline client/src/pages/Imprint.js:31]: Love the idea of using an imprint, with privacy information!

**Review by patrikglanemann — COMMENTED** [review]
LGTM!
When I go to account or imprint the arrow in the top right doesn't bring me back to ie the home page. I assume that is not implemented yet.?

File: client/src/pages/Imprint.js (hunk 1 above)
Comment (me-jterhorst) [reply client/src/pages/Imprint.js:31]: Thank you :)  

Yes the backButton isn't doing anything yet, but it's the first thing I am going to work on as soon as I have all screen layouts done

File: client/src/pages/Imprint.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import "./Imprint.css";
+import Card from "../components/Card";
+import BackButton from "../components/BackButton";
+
+export default function Imprint({ isLight }) {
+  return (
+    <main id="Imprint" className="card-screen">
```
Comment (namirsab) [inline client/src/pages/Imprint.js:7]: you shouldn't use the `id` here for styling purposes. Also, it'd be nice if you always follow the same convention: 
`className` of the root element of a component = the component name

**Review by namirsab — COMMENTED** [review]
@me-jterhorst 
Just a little comment. I know it's merged, but i couldn't resist ;)

**Pull request merged**